nupic/encoders/multi.capnp

```diff
@@ -0,0 +1,42 @@
+@0xf07c1f24bf12fe13;
+
+using import "adaptivescalar.capnp".AdaptiveScalarEncoderProto;
+using import "category.capnp".CategoryEncoderProto;
+using import "coordinate.capnp".CoordinateEncoderProto;
+using import "date.capnp".DateEncoderProto;
+using import "delta.capnp".DeltaEncoderProto;
+using import "geospatial_coordinate.capnp".GeospatialCoordinateEncoderProto;
+using import "log.capnp".LogEncoderProto;
+using import "pass_through.capnp".PassThroughEncoderProto;
+using import "random_distributed_scalar.capnp".RandomDistributedScalarEncoderProto;
+using import "scalar.capnp".ScalarEncoderProto;
+using import "sdrcategory.capnp".SDRCategoryEncoderProto;
+using import "sparse_pass_through_encoder.capnp".SparsePassThroughEncoderProto;
+
+# Next ID: 3
+struct MultiEncoderProto {
+  width @0 :UInt32;
```

Comment:
Can you drop width and just sum up the sub-encoder widths when you load them? That is how it creates width when constructed.